nano/lib/env.cpp

```diff
@@ -14,7 +14,8 @@ std::optional<std::string> nano::env::get (std::string_view name)
 	return std::nullopt;
 }
 
-std::optional<bool> nano::env::get_bool (std::string_view name)
+template <>
+std::optional<bool> nano::env::get (std::string_view name)
```

Comment:
Should the nano::env::get_int and nano::env::get_uint functions have similar specialisations or do they need to have their own name?

Reply:
I introduced and immediately replaced the `nano::env::get_int and nano::env::get_uint` variants with a generic implementation that uses `boost::lexical_cast` under the hood. So it's simply `nano::env::get<int>(...)` or `nano::env::get<unsigned>(...). I needed specialization for bool to enable it to parse additional values, for a better user experience when dealing with on/off values.